joomla/joomla-cms
# [com_messages] Add search tooltip

#### Summary of Changes

Simple PR to improve the search tooltip and element label (accessibility) in com_messages.
#### Testing Instructions

Code review, or:
1. Apply patch
2. Go to Components -> Messages , check the tooltip in the search field.
